cumulus_etl/etl/cli.py

```diff
@@ -68,15 +68,15 @@ def check_mstool() -> None:
         raise SystemExit(errors.MSTOOL_MISSING)
 
 
-def check_requirements() -> None:
+async def check_requirements(selected_tasks: Iterable[type[tasks.EtlTask]]) -> None:
     """
     Verifies that all external services and programs are ready
 
     May block while waiting a bit for them.
     """
-    nlp.check_ctakes()
-    nlp.check_cnlpt()
     check_mstool()
+    for task in selected_tasks:
+        await task.init_check()
```

Comment:
Because I the covid support services like ctakes/cnlpt are now study-specific, I figured it made sense to finally add the ability for the selected tasks to check for the services they'll need.

So now the covid tasks check for ctakes and the cnlpt models it needs.